chemprop/utils/utils.py

```diff
@@ -57,7 +57,13 @@ def make_mol(smi: str, keep_h: bool, add_h: bool) -> Chem.Mol:
     else:
         mol = Chem.MolFromSmiles(smi)
 
-    return Chem.AddHs(mol) if add_h else mol
+    if add_h:
+        mol = Chem.AddHs(mol)
+
+    if mol is None:
+        raise RuntimeError(f"SMILES {smi} is invalid! (RDKit returned None)")
```

Comment:
I think this needs to be moved before L60-61.

Reply:
Thanks, done in 39721e4ac899ddda056249957efac76a70a0c844